src/main/java/org/zalando/nakadi/domain/NakadiCursor.java

```diff
@@ -7,7 +7,7 @@
     public static final int VERSION_LENGTH = 3;
     public enum Version {
         ZERO("000"),
-        ONE("C00"),;
+        ONE("001"),;
```

Comment:
My problem now is that there's no space between versions ZERO and ONE. But you remember we have a "Plan B" how to rollout timelines without changing offsets for users. If we decide to apply that plan we will probably need some version between ZERO and ONE.
Wdyt?